plugin/trino-kafka/src/main/java/io/trino/plugin/kafka/KafkaMetadata.java

```diff
@@ -150,14 +150,15 @@ private Map<String, ColumnHandle> getColumnHandles(ConnectorSession session, Sch
         List<KafkaColumnHandle> topicColumnHandles = concat(keyColumnHandles, messageColumnHandles)
                 .collect(toImmutableList());
 
-        List<KafkaColumnHandle> internalColumnHandles = kafkaInternalFieldManager.getInternalFields().values().stream()
+        List<KafkaColumnHandle> internalColumnHandles = kafkaInternalFieldManager.getInternalFields().stream()
                 .map(kafkaInternalField -> kafkaInternalField.getColumnHandle(hideInternalColumns))
                 .collect(toImmutableList());
 
         Set<String> conflictingColumns = topicColumnHandles.stream().map(KafkaColumnHandle::getName).collect(toSet());
         conflictingColumns.retainAll(internalColumnHandles.stream().map(KafkaColumnHandle::getName).collect(toSet()));
         if (!conflictingColumns.isEmpty()) {
-            throw new IllegalArgumentException("Internal kafka fields names conflict with fields names from topic. "
+            throw new IllegalArgumentException("Internal kafka columns names conflict with columns names from the topic. "
```

Comment:
```suggestion
            throw new IllegalArgumentException("Internal Kafka column names conflict with column names from the table. "
```

Reply:
`columns names` vs `column names`

I always have internal discussion how to name such variables. And I still don't know how to do that properly. Are there any rules/best practices? Why only the second word should be in multiple form?

There are many columns and each of them has only one name. 
This is not true: only one column with multiple names.

cc @mosabua

Reply:
english is weird - the way I remember is you can have either world plural but not both - so `column names` or `column's names` AFAIK. 🤷

Reply:
single in both is correct..